docs/tutorials/coord_sys_tutorial.md

```diff
@@ -0,0 +1,211 @@
+
+### Lyft
+
+Lyft shares the same data format with NuScenes as far as coordinate system is involved.
+
+Please refer to the [official website](https://www.kaggle.com/c/3d-object-detection-for-autonomous-vehicles/data) for more information.
+
+### ScanNet
+
+The raw data of ScanNet is not point cloud but mesh. The sampled point cloud data is under our depth coordinate system. For ScanNet detection task, the box annotations are axis-aligned, and the yaw angle is always zero. Therefore the direction of the yaw angle in our depth coordinate system makes no difference regarding ScanNet.
+
+### SUN RGB-D
+
+The raw data of SUN RGB-D is not point cloud but RGB-D image. By back projection, we obtain the corresponding point cloud for each image, which is under our Depth coordinate system. However, the annotation is not under our system, hence needing conversion.
+
+For the conversion from raw annotation to annotation under our Depth coordinate system, please refer to [sunrgbd_data_utils.py](https://github.com/open-mmlab/mmdetection3d/blob/master/tools/data_converter/sunrgbd_data_utils.py).
+
+### S3DIS
+
+S3DIS shares the same coordinate system as ScanNet in our implementation. However, S3DIS is a segmentation-task-only dataset, thus no annotation is coordinate system sensitive.
+
+## Examples
+
+### Box conversion (between different coordinate systems)
+
+Take the conversion between our Camera coordinate system and LiDAR coordinate system as an example:
+
+First, for points and box centers, the coordinates before and after the conversion satisfy the following relationship:
+
+- <img src="https://render.githubusercontent.com/render/math?math=x_{LiDAR}=z_{camera}">
+- <img src="https://render.githubusercontent.com/render/math?math=y_{LiDAR}=-x_{camera}">
+- <img src="https://render.githubusercontent.com/render/math?math=z_{LiDAR}=-y_{camera}">
+
+Then, the box dimensions before and after the conversion satisfy the following relationship:
+
+- <img src="https://render.githubusercontent.com/render/math?math=dx_{LiDAR}=dz_{camera}">
+- <img src="https://render.githubusercontent.com/render/math?math=dy_{LiDAR}=dx_{camera}">
+- <img src="https://render.githubusercontent.com/render/math?math=dz_{LiDAR}=dy_{camera}">
+
+Finally, the yaw angle should also be converted:
+
+- <img src="https://render.githubusercontent.com/render/math?math=r_{LiDAR}=-\frac{\pi}{2}-r_{camera}">
+
+See the code [here](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/core/bbox/structures/box_3d_mode.py) for more details.
+
+### Bird's Eye View
+
+The BEV of a camera coordinate system box is <img src="https://render.githubusercontent.com/render/math?math=(x, y, dx, dy, -r)"> if the 3D box is <img src="https://render.githubusercontent.com/render/math?math=(x, y, z, dx, dy, dz, r)">. The inversion of the sign of the yaw angle is because the positive direction of the gravity axis of the Camera coordinate system points to the ground.
```

Comment:
I'm not sure but I think the coord of BEV is (x, z, dx, dz, -r) based on the [illustration](https://github.com/open-mmlab/mmdetection3d/blob/00da4257e6820db76a8e88b1a0bcf02c4adaf49c/resources/coord_sys_all.png) instead of (x, y, dx, dy, -r).

Reply:
You're right.